src/PheatureFlagsBundle.php

```diff
@@ -22,6 +22,7 @@ final class PheatureFlagsBundle extends Bundle
 {
     private const DEFAULT_CONFIG = [
         'driver' => 'inmemory',
+        'driver_options' => [],
```

Comment:
I think that the field isn't required here. It will not hurt, but not required, because we load the config from the "pheature_flags" extension, which loads the config using the `Pheature\Crud\Psr11\Toggle\ToggleConfig` class. The `DEFAULT_CONFIG` constant defines the minimum required fields to construct the `ToggleConfig` class.

As I say we can merge it because it makes the Bundle class more understandable. What do you think @xserrat and @ajgarlag?

Reply:
I'm wrong. It works because in runtime we always are loading the correct config via application config. Trying to unit test the Bundle class, I met the issue:

![image](https://user-images.githubusercontent.com/1093654/196763216-8bd0eed0-7913-4fc8-bd7a-963447eef183.png)
![image](https://user-images.githubusercontent.com/1093654/196763262-08fbf55e-e3f3-4362-9028-21ab121313fc.png)